<launch>
  <arg name='host' default='localhost'/>
  <arg name='port' default='2000'/>
  <arg name='timeout' default='10'/>
  <arg name="role_name" default="ego_vehicle" />

  <node pkg="carla_vertical_parking" type="carla_vertical_parking_node.py" name="carla_vertical_parking_node" output="screen">
    <param name="role_name" value="$(arg role_name)" />
    <param name="host" value="$(arg host)" />
    <param name="port" value="$(arg port)" />
    <param name="timeout" value="$(arg timeout)" />
  </node>


</launch>
